sbncode/CAFMaker/CMakeLists.txt

```diff
@@ -43,6 +43,8 @@ art_make_library( LIBRARY_NAME sbncode_CAFMaker
                   systematicstools::interface
                   systematicstools::interpreters
                   systematicstools::utility
+                  ${GENIE_LIB_LIST}
```

Comment:
Where is this variable defined? is it necessary?
You may also ask CMake to print its value (I think something like `message(STATUS "GENIE_LIB_LIST='${GENIE_LIB_LIST}'")`).
If this variable is empty or unnecessary, we should not put it (they often go viral).

Reply:
Thank you for finding that. It's not really a standard pattern (in fact, I question it), but it's valid.